docs/content/dagster-cloud/account/managing-users.mdx

```diff
@@ -1,668 +1,90 @@
 ---
-title: Managing users and permissions in Dagster Cloud | Dagster Docs
+title: Managing users in Dagster Cloud | Dagster Docs
 ---
 
-# Managing users and permissions in Dagster Cloud
+# Managing users in Dagster Cloud
 
 <Note>This guide is applicable to Dagster Cloud.</Note>
 
-Users in Dagster Cloud can be granted specific permissions what they can and cannot do in your organization. Granularly assigning permissions to users ensures that the right people have appropriate access to what they require in Dagster Cloud, and no more.
+Role-based access control (RBAC) enables you to grant specific permissions to users in your organization, ensuring that the right people have appropriate access to what they require in Dagster Cloud, and no more.
 
-In this guide, we'll cover the available user roles, how permissions are assigned, and how to add and remove users in your organization.
+In this guide, we'll cover how to add and remove users in your organization.
 
 ---
 
-## Understanding user roles
```

Comment:
This info (along with the permissions reference table) was moved to `/dagster-cloud/account/managing-users/managing-user-roles-permissions`